custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -125,12 +130,31 @@ def _init_mlops(self):
             self._mlops.set_model_id(self._params["model_id"])
 
         if self._supports_chat():
-            self._mlops.set_channel_config("spooler_type=API")
+            self._configure_mlops_for_chat()
         else:
             self._configure_mlops_for_non_chat()
 
         self._mlops.init()
 
+    def _configure_mlops_for_chat(self):
+        self._mlops.set_channel_config("spooler_type=API")
+
+        self._prompt_column_name = self._get_prompt_column_name()
+        logger.debug("Prompt column name: %s", self._prompt_column_name)
+
+    def _get_prompt_column_name(self):
+        if not self._params.get("deployment_id", None):
+            logger.error("No deployment id found while configuring mlops for chat.")
```

Comment:
Maybe also mention falling back to using default prompt column? So it could say something like:
```suggestion
            logger.error("No deployment ID found while configuring mlops for chat. Fallback to default prompt column name ('promptText')")
```

Reply:
Done